Engineering.md

```diff
@@ -189,6 +189,43 @@
 - I am becoming a trusted voice in Loan Market and my opinions and ideas seem to carry some weight.
 - I am mentoring another team member, deliberately helping them to round out their skill set.
 - I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles.
+---
+
+## Engineering Operations Lead
+> I am a well-rounded developer, architect and problem solver. I am trusted to set processes and practices that help engineers support our platforms established value while balancing engineering ideals with external pressures.
```

Comment:
Is architect a key part of the role? I think that's a loaded term so perhaps we should replace it with something else? e.g. analyst?

Reply:
I probably wouldn't have used that term either, I copied it from the principal engineer top level and I think the intended meaning is appropriate in both cases.  Maybe we should change it for both to be something around "solution designer" like - "I am a well-rounded developer, solution designer, problem solver and leader of people. I aspire to have engineering and behavioural excellence."?

Reply:
Yep I can live with that. Do you agree @adamchester-lmg ?

Reply:
Yes makes sense to me